generator/mysql/query_set.go

```diff
@@ -56,30 +29,33 @@ SELECT
 		) AS "dataType.Name",
 		IF (col.DATA_TYPE = 'enum', 'enum', 'base') AS "dataType.Kind",
 		col.COLUMN_TYPE LIKE '%unsigned%' AS "dataType.IsUnsigned"
-FROM
+FROM INFORMATION_SCHEMA.tables AS t
+LEFT OUTER JOIN
```

Comment:
It can be INNER JOIN. It is not possible to have a table without columns in MySQL.